### PR TITLE
feat: enable release of CLI distribution for Linux

### PR DESCRIPTION
**Pull request by aayushmau5**
**Description**

- This PR adds linux packaging and releasing for the CLI.
- Update README for installation instruction

**Related issue(s)**: https://github.com/asyncapi/cli/issues/303

### CODE REVIEW

**Comment by aayushmau5** [comment]
@derberg Some things remain/todo:

- First, we need to verify that the CI actually works. When I tryed installing the package locally, some additional packages were needed. So, we need to verify whether those package are pre-installed in the CI or not.
- Currently, I have only added `deb` releases, we need to support `.tar.gz` as well, i think.

File: scripts/linuxDebRename.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const { rename } = require('fs').promises;
```
Comment (derberg) [inline scripts/linuxDebRename.js:3]: There is already a lot of code that is in [macosPackRename.js](https://github.com/asyncapi/cli/blob/master/scripts/macosPackRename.js). Maybe merge both, and have one script to rename all distros? and `"pack:macos:rename": "node scripts/macosPackRename.js"` becomes `"pack:rename": "node scripts/releasePackagesRename.js"` ?

**Review by derberg — COMMENTED** [review]
There is an additional option from oclif to generate `.tar.gz`, you can try it out too -> https://github.com/oclif/oclif#oclif-pack-tarballs

File: .github/workflows/upload-release-assets.yml [hunk 2]
```diff
@@ -15,10 +15,10 @@ jobs:
...
       matrix:
         include:
-          # - os: ubuntu-latest
-          #   npm_script: pack:linux
-          #   dist_folder: deb
-          #   extension: tar.gz #or deb, we do not know yet
+          - os: ubuntu-latest
+            npm_script: pack:linux
+            dist_folder: deb
+            extension: deb #or deb, we do not know yet
```
Comment (aayushmau5) [inline .github/workflows/upload-release-assets.yml:21]: @derberg I'm not sure what this step does? What's the significance of `extension` here? 

Also, I guess I have to add another command to package for `tar.gz`, right?(and create a new step with `extension: tar.gz`)

Comment (derberg) [reply .github/workflows/upload-release-assets.yml:21]: it is used here `files: dist/${{ matrix.dist_folder }}/asyncapi.${{ matrix.extension }}` when pointing to binary file, and its extension

> Also, I guess I have to add another command to package for tar.gz, right?(and create a new step with extension: tar.gz)

yup cause I also guess you will not put dist into `deb` but something else

File: README.md [hunk 3]
```diff
@@ -67,6 +67,19 @@ curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.pkg
...
+
+You can install this CLI using `dpkg`, a package manager for debian.
+
+```bash
+# Download latest release. To download specific release, your link should look similar to https://github.com/asyncapi/cli/releases/download/v0.13.0/asyncapi.deb. All releases are listed in https://github.com/asyncapi/cli/releases
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.deb
+# Install AsyncAPI CLI
+sudo dpkg -i asyncapi.deb
+```
+
```
Comment (derberg) [inline README.md:82]: @aayushmau5 can you also add instruction for others, using `tar.gz`?

**Review by derberg — COMMENTED** [review]
@aayushmau5 perfect job, just left one comment regarding readme

**Comment by aayushmau5** [comment]
@derberg Added installation instruction for tar files. Let me know what you think.

**Review by derberg — APPROVED** [review]
@aayushmau5 thanks a 💯

**Pull request merged**